### PR TITLE
(0012247) Implement firewall image version parameter into firewall_instance resource

### PR DESCRIPTION
**Pull request by zhiqiangjin**
This will be available for controller 5.3.

### CODE REVIEW

File: aviatrix/resource_aviatrix_firewall_instance.go [hunk 1]
```diff
@@ -165,6 +172,7 @@ func resourceAviatrixFirewallInstanceRead(d *schema.ResourceData, meta interface
 	d.Set("firenet_gw_name", fI.GwName)
 	d.Set("firewall_name", fI.FirewallName)
 	d.Set("firewall_image", fI.FirewallImage)
+	d.Set("firewall_image", fI.FirewallImage)
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_firewall_instance.go]: firewall_image_version?

**Comment by arvindsree** [comment]
would it cause diff in existing deployments?

File: aviatrix/resource_aviatrix_firewall_instance.go (hunk 1 above)
Comment (zhiqiangjin) [reply aviatrix/resource_aviatrix_firewall_instance.go]: Redundant set. Will remove it.

**Comment by zhiqiangjin** [comment]
> would it cause diff in existing deployments?

backend only returns firewall image version when it was provided in creation. So, it will not cause diff in existing deployments.

**Review by arvindsree — APPROVED** [review]

**Pull request merged**